p2p/README.md

```diff
@@ -40,7 +40,7 @@ flowchart TD
 _DISCUSS(team): If you feel this needs a diagram, please reach out to the team for additional details._
 _TODO(olshansky, BenVan): Link to RainTree visualizations once it is complete._
 
-The `Network Module` is where RainTree (or the simpler basic approach) is implemented. See `raintree/network.go` for the specific implementation of RainTree, but please refer to the specifications for more details.
+The `Network Module` is where [RainTree](https://drive.google.com/file/d/1PaAWTx47nPSrM798lxfKv0GnhLthZLYm/view) (or the simpler basic approach) is implemented. See `raintree/network.go` for the specific implementation of RainTree, but please refer to the specifications for more details.
```

Comment:
Is google drive the best way to host this? Can't we upload to github

Reply:
On that note, why not IPFS :)

But yea, I believe we have a budget for `git lfs` so let's try to use that.

Will follow that pattern myself

Reply:
Fair enough guys. I have uploaded the pdf on this PR (Github will keep it stored unless it is deleted forcefully) and updated the link accordingly.

I didn't upload the pdf to git because we could end up having to download a lot of stuff locally but if you prefer that approach, just tell me where should we start adding pdfs and I'll commit to that approach.